.gitignore

```diff
@@ -11,3 +11,6 @@ cypress/downloads/
 /tests/.phpunit.result.cache
 dist/
 webpack-stats.json
+
+# js folder, to be updated wth --force
+/js
```

Comment:
This was an issue, weirdly, if you have any ignored file at least ONCE in the git history, it will suggest to be committed again. Which defeated the purpose of not suggesting to add the changed files. That what made me stop any further progress on the server PR. I wanted to make sure only CI was updating the js folder, and no one mistakenly added their own changes in a future PR :thinking:

Reply:
Seems like a good idea yes :)
You can even probably use the GH path trigger property for this

Reply:
Added as an adjustment to the node.yml file